packages/governance/src/types.js

```diff
@@ -454,7 +454,7 @@
  * @property {() => Instance} getGovernedContract
  * @property {(voteCounter: Instance) => Promise<boolean>} validateVoteCounter
  * @property {(regP: ERef<Instance>) => Promise<boolean>} validateElectorate
- * @property {(details: QuestionDetails) => boolean} validateTimer
+ * @property {(details: ParamChangeIssueDetails) => boolean} validateTimer
```

Comment:
> I think the more general type is correct. Why did you narrow it? validateTimer should work for any question.

Narrowed because it was overly constraining what was required for the param.
```
Argument of type 'ParamChangeIssueDetails' is not assignable to parameter of type 'QuestionDetails'.
  Type 'ParamChangeIssueDetails' is not assignable to type 'QuestionSpec<Issue>'.
    Types of property 'positions' are incompatible.
      Type 'ParamChangePositions' is missing the following properties from type 'Position[]': length, pop, push, concat, and 29 more.

39     E(governorPublic).validateTimer(details),
```

All it needs is the `closingRule` property:
https://github.com/Agoric/agoric-sdk/blob/8921f59/packages/governance/src/contractGovernor.js#L213-L219

I've changed it to take and check that specifically: d42b58e3b